crabs/detector/utils/detection.py

```diff
@@ -242,3 +243,74 @@ def slurm_logs_as_artifacts(logger: MLFlowLogger, slurm_job_id: str):
             logger.run_id,
             f"{log_filename}.{ext}",
         )
+
+
+def bbox_tensors_to_COCO_dict(
```

Comment:
Nice! Do you plan to use this function in the code or just for testing? If just for the test, then maybe you can move this to fixture folder for test? But I think this might

Reply:
Yeah agreed! Initially I had it with the tests, but then I moved it to utils thinking that:
- It may be useful in other scenarios (for example, if we implement an option to export detections in VIA format  #231), and
- I would like to test it well at some point. I feel it is more likely to be left untested if I leave it inside tests as a helper function.